activity: add fallback isolation level for statements not in transaction

jasonlmfong opened the pull request:
The change is achieved by retrieving the session default isolation level

closes: [cockroachdb#153931](cockroachdb#153931)
Epic: None
Release note: None

rafiss reviewed (approved):
thanks for the fix! feel free to merge after addressing my comment

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/154027#-:-O_wR7caBVv89aZqM1d7:blju4ma)* status: <span>:shipit:</span> complete! 0 of 0 LGTMs obtained (waiting on @alyshanjahani-crl)

___
*[`-- commits` line 8 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/154027#-O_wR-Dz79kz39mQr_Sn:-O_wR-Dz79kz39mQr_So:b-o60jhk):*
nit: we should not have a release note here, as the bug being fixed was never released to customers.


<!-- Sent from Reviewable.io -->

jasonlmfong commented:
Thanks, will fix that

jasonlmfong committed:
activity: add fallback isolation level for statements not in transaction

The change is achieved by retrieving the session default isolation level

closes: cockroachdb[#153931](https://github.com/cockroachdb/cockroach/issues/153931)
Epic: None
Release note: None
